packages/@cdktf/cli-core/.eslintignore

```diff
@@ -1,2 +1,5 @@
 # ignore build output
-*.d.ts
\ No newline at end of file
+*.d.ts
+
+# ignore templates, since they're not valid TypeScript
+templates
```

Comment:
Now that we're using syntax like:

```
export class {{basePascal}}Stack extends TerraformStack {
```

We have invalid TS, so eslint blows up.